openedx_webhooks/tasks/pr_tracking.py

```diff
@@ -259,7 +261,7 @@ def desired_support_state(pr: PrDict) -> Optional[PrDesiredInfo]:
             if map_1_2 is not None:
                 desired.jira_extra_fields["Platform Map Area (Levels 1 & 2)"] = map_1_2
     elif desired.is_ospr:
-        if state == "open":
+        if state in ["open", "reopened"]:
```

Comment:
Does this re-add the welcome comment on a reopen?

Reply:
No.  Here's a PR where I did manual testing:  https://github.com/nedbat/webhook-testing/pull/101